docs/setup-cluster/k8s/setup-aks-cluster.rst

```diff
@@ -78,9 +78,8 @@ command:
  Enable GPU Support
 ********************
 
-To allow the AKS cluster to recognize GPU hardware resources, refer to the instructions provided by
-Azure on the `Install NVIDIA Device Plugin
-<https://docs.microsoft.com/en-us/azure/aks/gpu-cluster#install-nvidia-device-plugin>`_ tutorial.
```

Comment:
This link seems fine. Maybe the link-checker freaked out? Leave this link in for now.

Reply:
yes the link seems fine